src/routes/transactions/__tests__/controllers/list-queued-transactions-by-safe.transactions.controller.spec.ts

```diff
@@ -468,4 +468,111 @@ describe('List queued transactions by Safe - Transactions Controller (Unit)', ()
         });
       });
   });
+
+  it('should get an "untrusted" queue', async () => {
+    const chainId = faker.string.numeric();
+    const safeAddress = faker.finance.ethereumAddress();
+    const chainResponse = chainBuilder().build();
+    const contractResponse = contractBuilder().build();
+    const safeResponse = safeBuilder()
+      .with('address', safeAddress)
+      .with('nonce', 1)
+      .build();
+    const safeAppsResponse = [
+      safeAppBuilder()
+        .with('url', faker.internet.url({ appendSlash: false }))
+        .with('iconUrl', faker.internet.url({ appendSlash: false }))
+        .with('name', faker.word.words())
+        .build(),
+    ];
+    const transactions: MultisigTransaction[] = [
+      multisigToJson(
+        multisigTransactionBuilder()
+          .with('safe', safeAddress)
+          .with('isExecuted', false)
+          .with('safeTxHash', faker.finance.ethereumAddress())
```

Comment:
The `safeTxHash` is 32 characters long, but `multisigTransactionBuilder` generates this automatically.

```suggestion
```